libflux/flux-core/src/formatter/mod.rs

```diff
@@ -1914,28 +1914,28 @@ fn get_precedences(parent: &Node, child: &Node) -> (u32, u32) {
     let pvp: u32 = match parent {
         Node::BinaryExpr(p) => Operator::new(&p.operator).get_precedence(),
         Node::LogicalExpr(p) => Operator::new_logical(&p.operator).get_precedence(),
-        Node::UnaryExpr(p) => Operator::new(&p.operator).get_precedence(),
-        Node::FunctionExpr(_) => 3,
-        Node::PipeExpr(_) => 2,
+        Node::UnaryExpr(p) => Operator::new(&p.operator).get_precedence_unary_operator(),
```

Comment:
This is more of a nit than anything, re: consistency.

I notice in the case of logical expressions, we use `Operator::new_logical()` which sets a bool field inside the `Operator` struct, which is then used to branch inside `Operator::get_precedence()`.

I wonder if that design could work for the unary case rather than delegating to a new method. Are there barriers to this? Maybe it just makes the flow more complicated.

Reply:
@onelson That makes sense to follow the same design here as well. I will do the changes.